ChangeFeedLabSolution/ChangeFeedProcessor.cs

```diff
@@ -50,25 +51,20 @@ public static class ChangeFeedProcessor
             string eventHubNamespaceConnection = ConfigurationSettings.AppSettings["EventHubNamespaceConnection"];
 #pragma warning restore CS0618 // Type or member is obsolete
 
-            // Build connection string to access event hub within event hub namespace.
-            EventHubsConnectionStringBuilder eventHubConnectionStringBuilder =
-                new EventHubsConnectionStringBuilder(eventHubNamespaceConnection)
-                {
-                    EntityPath = EventHubName
-                };
-
-            // Create event hub client to send change feed events to event hub.
-            EventHubClient eventHubClient = EventHubClient.CreateFromConnectionString(eventHubConnectionStringBuilder.ToString());
-
-            // Iterate through modified documents from change feed.
-            foreach (var doc in documents)
+            // Create producer client to send change feed events to event hub.
+            await using (var producer = new EventHubProducerClient(eventHubNamespaceConnection, EventHubName))

```

Comment:
This isn't the correct pattern to use.   The producer client should be created as a `static` member of the class, following the pattern shown in the `DocumentClient` sippet, [here](https://docs.microsoft.com/en-us/azure/azure-functions/manage-connections#documentclient-code-example-c).